packages/graphql/schemas/schema.graphql

```diff
@@ -1140,9 +1141,6 @@ type Query {
   """The currently opened project"""
   currentProject: CurrentProject
 
-  """The mode the interactive runner was launched in"""
-  currentTestingType: TestingTypeEnum
```

Comment:
Is this still available somehow via GraphQL? Or do we we not need it anymore?

Reply:
Edit: I think it's here but on `currentProject`: https://github.com/cypress-io/cypress/pull/20558/files#diff-99636480d4ea376f639524746fcddf8126cd1ad0abf0f5a2b570d6bb56085722R99

Reply:
Yeah we had this duplicated on query, updated the PR description to call this out